en/docs/guides/identity-federation/configure-shibboleth-idp-as-a-trusted-identity-provider.md

```diff
@@ -0,0 +1,427 @@
+# Configure Shibboleth IdP as a Trusted Identity Provider
```

Comment:
Shall we check with Hasanthi if these pages have updated content? If it is very obviously outdated, we might be better off not including it at all

Reply:
There will be a developer review once we are done with moving the content so I guess we can remove these in bulk.